FIAT/__init__.py

```diff
@@ -29,7 +28,7 @@
 from FIAT.tensor_product import TensorProductElement
 from FIAT.enriched import EnrichedElement
 from FIAT.discontinuous import DiscontinuousElement
-from FIAT.trace_hdiv import HDivTrace
+from FIAT.hdiv_trace import TraceHDiv
```

Comment:
Sure. Or we can give it a completely different name if you prefer. Like "TraceElement."